train.py

```diff
@@ -11,12 +11,12 @@
 from src.pipeline_funcs import train_model
 
 selected_bbox = bboxes["Sudan_Blue_Nile"]
-train_datasets = [datasets[1].dataset]  # [d.dataset for d in datasets]
+# train_datasets = [datasets[1].dataset]  # [d.dataset for d in datasets]
```

Comment:
Looks like CI training time did not increase too much, I am fine with removing this line completely